src/soilId/soilIdFragments.ts

```diff
@@ -29,6 +29,7 @@ export const soilInfo = /* GraphQL */ `
       taxonomySubgroup
       description
       fullDescriptionUrl
+      management
```

Comment:
reminder: the plan is to remove this from the backend before full release, so we should make sure this doesn't end up on any end users' devices!